### PR TITLE
the second part of Accont page

### PR DESCRIPTION
**Pull request by husamkamal**
Relates #12

### CODE REVIEW

**Review by IsraaAmani — DISMISSED** [review]

File: Account/index.html [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://pro.fontawesome.com/releases/v5.10.0/css/all.css" integrity="sha384-AYmEC3Yw5cVb3ZcuHtOA93w35dYTsvhLPVnYs9eStHfGJvOvKxVfELGroGkvsg+p" crossorigin="anonymous"/>
+    <title>FoodApp</title>
+</head>
+<body>
+    <main>
+        <section class="button">
```
Comment (yaqootturman) [inline Account/index.html:13]: use a descriptive class name, pleasse!

File: Account/index.html [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+            <button >Posts</button>
+            <button>Reviews</button>
+        </section>
+        <hr class="hr1">
+        <section class="aconnt-fram">
+            <article class="div1">
```
Comment (yaqootturman) [inline Account/index.html]: Its not cool way to use order for naming such as div1, div2 ....
and same for others hr1....

File: Account/style.css [hunk 3]
```diff
@@ -0,0 +1,111 @@
+body{
```
Comment (yaqootturman) [inline Account/style.css:1]: Do you need to add responsive part?

**Review by yaqootturman — CHANGES_REQUESTED** [review]

File: Account/style.css (hunk 3 above)
Comment (husamkamal) [reply Account/style.css:1]: I have added it

File: Account/style.css [hunk 4]
```diff
@@ -0,0 +1,171 @@
...
+    width: 8rem;
+    left: 3rem;
+    }
+    article{
+        margin-bottom: 3rem;
+    }
+
+
+}
+@media  (min-width:500px) and (max-width:599) {
```
Comment (yaqootturman) [inline Account/style.css]: why using this width?
we have tablet and desktop width already

File: Account/style.css [hunk 5]
```diff
@@ -0,0 +1,171 @@
...
+        top: .9rem;
+        width:6rem ;
+    }
+    article{
+        margin-bottom: 2rem;
+    
+    }
+    
+}
+@media  (min-width:800px) and (max-width:1024px) {
```
Comment (yaqootturman) [inline Account/style.css]: the images become pixel and not responsive in desktop and tablet size

File: Account/index.html (hunk 1 above)
Comment (yaqootturman) [reply Account/index.html:13]: same

**Review by yaqootturman — CHANGES_REQUESTED** [review]

**Review by IsraaAmani — DISMISSED** [review]

**Review by yaqootturman — APPROVED** [review]

**Review by IsraaAmani — APPROVED** [review]

**Pull request merged**